.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-- 6.2.0
```

Comment:
https://eslint.org/docs/user-guide/migrating-to-6.0.0#nodejs-6-is-no-longer-supported